rosidl_cli/rosidl_cli/entry_points.py

```diff
@@ -12,38 +12,41 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import importlib.metadata as importlib_metadata
 import logging
-
-try:
-    import importlib.metadata as importlib_metadata
-except ModuleNotFoundError:
-    import importlib_metadata
+import sys
+from typing import Any, Dict, List, Optional, Tuple, Union
 
 
 logger = logging.getLogger(__name__)
 
 
-def get_entry_points(group_name, *, specs=None, strict=False):
+def get_entry_points(group_name: str, *, specs: Optional[List[str]] = None, strict: bool = False
+                     ) -> Dict[str, importlib_metadata.EntryPoint]:
     """
     Get entry points from a specific group.
 
-    :param str group_name: the name of the entry point group
-    :param list specs: an optional collection of entry point names to retrieve
-    :param bool strict: whether to raise or warn on error
+    :param group_name: the name of the entry point group
+    :param specs: an optional collection of entry point names to retrieve
+    :param strict: whether to raise or warn on error
     :returns: mapping from entry point names to ``EntryPoint`` instances
-    :rtype: dict
     """
     if specs is not None:
-        specs = set(specs)
+        specs_set = set(specs)
+    else:
+        specs_set = None
```

Comment:
this is the fix for the regression reported with https://github.com/osrf/buildfarm-tools/issues/99, and this is the only difference from https://github.com/ros2/rosidl/pull/826/commits